dev/fattest.simplicity/src/componenttest/rules/repeater/CheckpointRule.java

```diff
@@ -447,8 +450,12 @@ private void configureBootStrapProperties() throws Exception {
             bootStrapProperties.remove("websphere.java.security.norethrow");
             bootStrapProperties.remove("websphere.java.security.unique");
 
-            try (OutputStream out = new FileOutputStream(server.getServerBootstrapPropertiesFile().getAbsolutePath())) {
-                bootStrapProperties.store(out, "");
+            try (BufferedWriter writer = new BufferedWriter(new FileWriter(server.getServerBootstrapPropertiesFile().getAbsolutePath()))) {
```

Comment:
What issue is this change fixing?

Reply:
The previous code was escaping the special characters so were getting multiple backslashes causing errors. I tried to write the reason in the commit message. :)